### ISSUE TITLE
build for 2015 & 2016

### ISSUE THREAD

**Issue by redpawfx** [redpawfx/partio#39]

**Comment by ytsejam** [comment]
Hello. any news?
Snap to point cloud would be great.

**Comment by jasonhuang1115** [comment]
indeed, would love to see a build for 2015 and the feature to snap to points.

**Comment by redpawfx** [comment]
Coming soon!   Win and  Linux versions..   
@ytsejam,  I have some other ideas for the snap to point cloud thing.

**Comment by jasonhuang1115** [comment]
Awesome! Thanks for the heads-up.

- Jason

**Comment by ytsejam** [comment]
@redpawfx, it sound good, tell me. I 'm using lot of procedures, any other things would be appreciated.

**Comment by ytsejam** [comment]
i can't wait. @redpawfx this news make me crazy

**Comment by ytsejam** [comment]
Hello. Any news?
